### ISSUE TITLE
Give host a warning when he is about to close his own server 

### ISSUE THREAD

**Issue by Micr0Bit** [OpenRA/OpenRA#12039]
some hosts dont understand ... that they are closing the game for everyone ... when they quit 

a warning would be nice ... having to "double"-accept when quitting a hosted game ... as host-player 

(this of course isnt an issue when having a dedicated server running)

**Comment by pchote** [comment]
Makes sense, we already do this when force-starting the game will kick players who don't have the map.  It should only show the dialog if there are other clients in the server.
